recipes/ckanbatch-configure.rb

```diff
@@ -74,7 +74,7 @@
     group "root"
 end
 
-file "/etc/cron.daily/solr-reindex" do
+file "/etc/cron.hourly/solr-reindex" do
     content "/usr/local/bin/pick-job-server.sh && #{ckan_cli} search-index rebuild -ieo >/dev/null 2>&1\n"
```

Comment:
``-ieo`` what does this do again?

Reply:
`-i`: Continue on error.
`-e`: Commit changes after each update, instead of in a batch.
`-o`: Only update missing packages.

Reply:
> this does not clean up 'deleted'

True, but it's no worse than the current state. We would have to run a full hours-long rebuild from scratch to fix that.